index.html

```diff
@@ -60,43 +79,32 @@ <h3 class="job-title">
                 <h2 class="recentWorks">
```

Comment:
- [ ] As written in the project requirements. I kindly suggest that you ensure that when a project is hovered on with the mouse:
-  the description and dark opacity disappear
-  and a See Project button appears to the bottom of the image and takes the whole image width.
-   Your `See Project` button should link to the Live Demo of the project or to the GitHub repository.
-   The project text should take up no more than 3 lines of text. Your project text currently takes up 4 lines of text.

| Your Design  |Template Design |
| ------------- | ------------- |
| <img width="865" alt="image" src="https://user-images.githubusercontent.com/52934128/229117140-1fb183e2-e817-4896-bfdf-5726c602a85f.png"> | ![image](https://user-images.githubusercontent.com/52934128/178112550-f1b33dc5-838e-4251-898a-ea299dc20431.png) |